### ISSUE TITLE
Error compiling module for android

### ISSUE THREAD

**Issue by PerduGames** [godotengine/godot#9737]
Debian, Godot source Git

I try to compile with the command below:

`scons platform=android target=release
`
I'm trying to compile the APKS. Using this module and am getting this error:

https://github.com/kloder-games/godot-admob

```
scons: Reading SConscript files ...
sh: 1: i586-mingw32msvc-gcc: not found
sh: 1: i686-w64-mingw32-gcc: not found
UnboundLocalError: local variable 'can_vectorize' referenced before assignment:
  File "/home/user/godot-master-3/SConstruct", line 273:
    detect.configure(env)
  File "./platform/android/detect.py", line 95:
    if (can_vectorize):

```
Why am I compiling on the git version? Because I need to use multiplayer, I'm at the same time studying Enet and trying to create a multiplayer module for godot, but the time is running, I would like to try this new API before continuing to write. And I need this module (admob) that is already integrated with my app. So what I thought, I'm going to try to compile it first, if it works out and everything works, I start migrating my project to the new godot.

**Comment by vnen** [comment]
Pretty sure this was fixed recently. Can you try to update your local clone?

**Comment by PerduGames** [comment]
I saw the fix here, I will update and test, anyway I wrote again, now I will finish, I summarized the maximum I gave to use what I want, so I think of ending with a simple P2P module. Finishing it, I'll make it available in open assets. I ended up enjoying this Enet, well organized documentation, I saw that you can do a lot with it.

**Comment by akien-mga** [comment]
Duplicate of fixed #9723.
